scripts/west_commands/runners/stm32cubeprogrammer.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) 2020 Teslabs Engineering S.L.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Runner for flashing with STM32CubeProgrammer CLI, the official programming
+   utility from ST Microelectronics.
+"""
+
+from pathlib import Path
+import platform
+import os
+import shlex
+from typing import TYPE_CHECKING, List, Optional, ClassVar, Dict
+
+from runners.core import ZephyrBinaryRunner, RunnerCaps
+
+if TYPE_CHECKING:
+    import argparse
+
+    from runners.core import RunnerConfig
```

Comment:
Why is this under TYPE_CHECKING?

It isn't a performance optimization AFAICT. Both argparse and runners.core have already been imported, so the imports in this file will hit the sys.modules cache and return immediately. Is the linter complaining without the `if` somehow?

Reply:
Good catch, both will already be imported.

Reply:
fixed